### ISSUE TITLE
Select docker tags when creating oneclick-app

### ISSUE THREAD

**Issue by danielivert** [githubsaturn/captainduckduck#134]
It would be nice to be able to select which version will installed as well.

**Comment by j7an** [comment]
Another possible option is to allow platform specific templates e.g. PHP, Python, Node, etc. with platform version e.g. PHP 7.2 so that if a user would like to install an app and its not implemented as a oneclick, then just select one of the template to install manually.

**Comment by githubsaturn** [comment]
@j7an I am not too sure if I understand your comment correctly.

But if you're talking about being able to use the platform version for simple captain-definition` files, they are already supported: https://github.com/githubsaturn/captainduckduck/wiki/Captain-Definition-File

Otherwise, if you're referring to one-click apps, they usually have their own image which is not necessarily built on any particular images, instead, they are customized images. For example, [this](https://github.com/docker-library/mysql/blob/6c414e7f38c2079c7193beae5dc7c34ee46cd6e7/8.0/Dockerfile) is the Dockerfile for MySql. Having said this, one-click apps are nothing more than some convenience helpers. For example, if you want to deploy a particular version of MySQL, say 5.7.21, rather than the "lastest" version that Captain deploys, here are the steps:
- Create a new app via Captain dashboard and check the "has persistent data" checkbox.
- Create a captain-definition file with this content:
```
{
  "schemaVersion": 1,
  "dockerfileLines": [
    "FROM mysql:5.7.21"
  ]
}
```
- Convert the captain-definition file into a tar file
- Upload (deploy) the tar file via Captain dashboard as your MySql image.
- Set the persistent volumes according to https://hub.docker.com/_/mysql/

**Comment by j7an** [comment]
My idea is that instead of having to wait for you to create an oneclick template, there could also be oneclick platform templates e.g. a PHP app template. Allow volume mapping via the platform template setup. Finally just upload files to the mapped directory on the host to install the app.

It's exactly as you've described for the steps in your bulleted list. Just another way for those that aren't devs wanting something simple.

**Comment by githubsaturn** [comment]
Ability to add specific tags added in https://github.com/githubsaturn/captainduckduck/commit/07a3cf3960eb512eba524821e669a913269d0f88

Will be released in v0.7 in the upcoming week.
